downloader.py

```diff
@@ -228,7 +230,9 @@ def get_authentication_token() -> str:
             "encrypted_key": rsa_encrypted_aes_key,
             "iv": iv,
         }
-        response = requests.post(url=BASE_URL + "/" + CONFIG.AUTH_ROUTE, data=auth_body)
+        print(BASE_URL)
```

Comment:
Could we remove this print statement?

Reply:
Sure, it was only introduced for debugging purposes.